packages/clerk-js/src/ui/utils/errorHandler.ts

```diff
@@ -15,10 +15,17 @@ function setFieldErrors(fieldStates: Array<FormControlState<string>>, errors: Cl
   }
 
   fieldStates.forEach(field => {
-    const error = errors.find(err => {
+    let buildErrorMessage = field?.buildErrorMessage;
+
+    if (!buildErrorMessage) {
+      buildErrorMessage = (err: ClerkAPIError[]) => err[0];
```

Comment:
Just to confirm, is this how we support backwards compatibility with single error fields ?

Reply:
Let's create a function called `getFirstError` or similar to make this more explicit :)